lib/syscall_shim/uk_syscall6.awk

```diff
@@ -7,13 +7,12 @@ BEGIN {
 
 	printf "long uk_syscall6(long nr, "
 	for (i = 1; i < max_args; i++)
-		printf "long arg%d, ",i
-	printf "long arg%d)\n{\n", max_args
+		printf "long arg%d __maybe_unused, ",i
+	printf "long arg%d __maybe_unused)\n{\n", max_args
+	printf "\tlong ret;\n\n"
```

Comment:
```suggestion
	print "\tlong ret;\n\n"
```

Reply:
Updated. Please note that `print` is also issueing a newline while `printf` is not doing this with awk 😉